CHANGES.rst

```diff
@@ -8,6 +8,9 @@ Changelog
 
 .. towncrier release notes start
 
+- Document next previous feature.
```

Comment:
use towncrier. use one PR not two (merge the new PR into this one once the docs are ready).